network/src/commonMain/kotlin/com/wire/kalium/network/api/v0/authenticated/TeamsApiV0.kt

```diff
@@ -28,21 +28,6 @@ internal class TeamsApiV0 internal constructor(
             httpClient.get("$PATH_TEAMS/$teamId")
         }
 
-    override suspend fun getTeams(size: Int?, option: TeamsApi.GetTeamsOption?): NetworkResponse<TeamsApi.TeamsResponse> =
```

Comment:
This part of the API is deprecated and not used in kalium anywhere, so I opted to delete it instead of keeping support for something that we are not using